acapy_agent/resolver/did_resolver.py

```diff
@@ -56,7 +56,9 @@ async def _resolve(
         if isinstance(did, DID):
             did = str(did)
         else:
-            DID.validate(did)
+            if not DID.is_valid(did) and not DIDUrl.is_valid(did):
```

Comment:
This allows for the DIDUrl to be resolved. I admittedly don't have a lot of knowledge here.